libnetdata/ebpf/ebpf.c

```diff
@@ -242,16 +242,18 @@ char *ebpf_kernel_suffix(int version, int isrh)
 {
     if (isrh) {
         if (version >= NETDATA_EBPF_KERNEL_4_11)
-            return "4.18.0";
+            return "4.18";
         else
-            return "3.10.0";
+            return "3.10";
     } else {
-        if (version >= NETDATA_EBPF_KERNEL_4_17)
-            return "5.4.20";
+        if (version >= NETDATA_EBPF_KERNEL_5_10)
+            return "5.10";
+        else if (version >= NETDATA_EBPF_KERNEL_4_17)
+            return "5.4";
         else if (version >= NETDATA_EBPF_KERNEL_4_15)
-            return "4.16.18";
+            return "4.16";
         else if (version >= NETDATA_EBPF_KERNEL_4_11)
-            return "4.14.171";
+            return "4.14";
```

Comment:
How come we removed the patch versions? My recollection is, that since ebpf does not use formal APIs even a patch version change can possibly change internal data structures and function calls.

Reply:
This change  is affecting the current name of the `eBPF` program, it does not affect the compilation itself, because this happens in another repository. If there is a change that force us to create a new version for a specific patch, we will have two different eBPF programs and the suffix will be different for both.